### PR TITLE
Use API to fetch post body and don't limit to location.hostname URLs

### PR DESCRIPTION
**Pull request by double-beep**
Show the content of any SE post, even from a different site.

### CODE REVIEW

File: sox.features.info.json [hunk 1]
```diff
@@ -431,7 +431,8 @@
             "extended_description": "Adds a button next to links to posts on the same site that expand to show the post inline",
```
Comment (shu8) [inline sox.features.info.json]: Could you update this description as well, to say it now works with posts on any other SE site?

Comment (double-beep) [reply sox.features.info.json]: Ok, done!

File: sox.features.js [hunk 2]
```diff
@@ -1669,9 +1669,21 @@
           $(this).addClass('expander-arrow-small-show');
           const $that = $(this);
           const id = getIdFromUrl($(this).next().attr('href'));
-          $.get(location.protocol + '//' + sox.site.url + '/posts/' + id + '/body', d => {
-            const div = '<div class="linkedPostsInline-loaded-body-sox">' + d + '</div>';
-            $that.next().after(div);
+          var url = $(this).next().attr('href');
```
Comment (shu8) [inline sox.features.js]: Please could you change this `var` to a `const` (or `let` but I don't think this is changing, so const might be better!)

**Comment by shu8** [comment]
Thanks for the change! Merged 😄

**Pull request merged**